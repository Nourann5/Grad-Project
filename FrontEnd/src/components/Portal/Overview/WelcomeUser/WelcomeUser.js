import React from 'react'
import styles from './WelcomeUser.module.css'
import {ReactComponent as StudentIcon} from 'assets/icons/studentIcon.svg'
import {ReactComponent as TeacherIcon} from 'assets/icons/teacherIcon.svg'
import {ReactComponent as SchoolIcon} from 'assets/icons/schoolIcon.svg'
import {ReactComponent as CenterIcon} from 'assets/icons/centerIcon.svg'
import {ReactComponent as CenterAcademy} from 'assets/icons/centerAcademy.svg'
import {ReactComponent as ColleageIcon} from 'assets/icons/colleageIcon.svg'
import {ReactComponent as UniversityIcon} from 'assets/icons/universityIcon.svg'
import {ReactComponent as MembersPortalIcon} from 'assets/icons/membersPortal.svg'
function WelcomeUser() {
  return (
    <div className={styles['overview__welcome-user']}>
        <div className={styles['overview__welcome-user-type-wrapper']}>
          <MembersPortalIcon className={styles['overview__welcome-user-type-icon']}/>
        </div>
        <div className={styles['overview__welcome-user-info']}>
          <h2 className={styles['overview__welcome-user-info-title']}>Teacher Portal</h2>
          <h3 className={styles['overview__welcome-user-info-welcome']}>Welcome ali mohammed In your dashboard</h3>
        </div>
        <div>

        </div>
    </div>

  )
}

export default WelcomeUser