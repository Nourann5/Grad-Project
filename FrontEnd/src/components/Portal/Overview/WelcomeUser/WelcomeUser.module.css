.overview__welcome-user{
    padding:20px;
    width:100%; 
    background: linear-gradient(97deg, var(--main-color) 0%, #9100FF 100%);
    margin-bottom:30px;
    display: flex;
    align-items: center;
    border-radius: 10px;
}
.overview__welcome-user-type-wrapper{
    padding: 20px;
    background-color: white;
    border-radius: 50%;
    margin-right: 20px;
}
.overview__welcome-user-type-icon{

}
.overview__welcome-user-info{

}
.overview__welcome-user-info-title{
    color: white;
    font-size: 24px;
    margin-bottom: 3px;
}
.overview__welcome-user-info-welcome{
    color: rgb(237, 235, 235);
    font-size: 16px;
    margin-bottom: 0;
}