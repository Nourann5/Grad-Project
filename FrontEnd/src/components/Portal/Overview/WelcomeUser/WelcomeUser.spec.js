import {render , screen} from '@testing-library/react'
import { BrowserRouter, Route } from 'react-router-dom'
import WelcomeUser from './WelcomeUser'

describe('Test the Welcome User', () => { 
    it('heading exists',()=>{
        const tree =  render(
            <BrowserRouter>
                <WelcomeUser/>
            </BrowserRouter>
        )
        expect(tree).toMatchSnapshot()
    })
 })