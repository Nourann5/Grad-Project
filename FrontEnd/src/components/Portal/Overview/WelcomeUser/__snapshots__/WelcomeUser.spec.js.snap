// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Welcome User heading exists 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="overview__welcome-user"
      >
        <div
          class="overview__welcome-user-type-wrapper"
        >
          <svg
            class="overview__welcome-user-type-icon"
          >
            membersPortal.svg
          </svg>
        </div>
        <div
          class="overview__welcome-user-info"
        >
          <h2
            class="overview__welcome-user-info-title"
          >
            Teacher Portal
          </h2>
          <h3
            class="overview__welcome-user-info-welcome"
          >
            Welcome ali mohammed In your dashboard
          </h3>
        </div>
        <div />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="overview__welcome-user"
    >
      <div
        class="overview__welcome-user-type-wrapper"
      >
        <svg
          class="overview__welcome-user-type-icon"
        >
          membersPortal.svg
        </svg>
      </div>
      <div
        class="overview__welcome-user-info"
      >
        <h2
          class="overview__welcome-user-info-title"
        >
          Teacher Portal
        </h2>
        <h3
          class="overview__welcome-user-info-welcome"
        >
          Welcome ali mohammed In your dashboard
        </h3>
      </div>
      <div />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
